openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -99,8 +101,39 @@ def execute(self):
 
         app_arguments = self._get_launch_arguments(data_to_script)
 
+        # fix project data permission issue
+        self._fix_permissions(project_name, volume_name)
+
         self.launch_context.launch_args.extend(app_arguments)
 
+    def _fix_permissions(self, project_name, volume_name):
+        """Work around for project data permissions
+
+        Reported issue: when project is created locally on one machine,
+        it is impossible to migrate it to other machine. Autodesk Flame
+        is crating some unmanagable files which needs to be opened to 0o777.
+
+        Args:
+            project_name (str): project name
+            volume_name (str): studio volume
+        """
+        dirs_to_modify = [
```

Comment:
Not sure if flame can be opened on windows?
```suggestion

        # Skip if 'chmod' is not available
        if not hasattr(os, "chmod"):
            return

        dirs_to_modify = [
```

Reply:
No it cannot. It is only Linux and Macos..